python/delta/tables.py

```diff
@@ -110,15 +181,21 @@ def merge(self, source, condition):
         jbuilder = self._jdt.merge(source._jdf, self._condition_to_jcolumn(condition))
         return DeltaMergeBuilder(self._spark, jbuilder)
 
+    @since(0.4)
     def vacuum(self, retentionHours=None):
         """
         Recursively delete files and directories in the table that are not needed by the table for
         maintaining older versions up to the given retention threshold. This method will return an
-        empty DataFrame on successful completion. This uses the default retention period of 7
-        days.
-        :param retentionHours:
-        :return: DataFrame of the history
+        empty DataFrame on successful completion.
+
+        Example::
 
+            deltaTable.vacuum()     # vacuum files not required by versions more than 7 days old
+
+            deltaTable.vacuum(100)  # vacuum files not required by versions more than 100 hours old
+
+        :param retentionHours: Optional number of hours retain history. If not specified, then the
+                               default retention period of 7 days will be used.
```

Comment:
I get why it's this way, but it's still super confusing that the param is in hours and the default is days. Maybe say it in both hours and days? "default retention period of 168 hours (7 days) will be used"?

Reply:
done.